cmd/analyzer/parse_args.go

```diff
@@ -77,7 +111,12 @@ func ParseInArgs(cmdlineArgs []string) (*InArgs, error) {
 	args.Debug = flagset.Bool("debug", false, "run in debug mode")
 	args.Version = flagset.Bool("version", false, "prints the release version number")
 
-	err := flagset.Parse(cmdlineArgs)
+	err := parseCmdLine(cmdlineArgs)
```

Comment:
when running with `-h` getting:
```
$ ./bin/vpcanalyzer -h
2023/12/19 12:19:24 error parsing arguments: flag not supported: -h. exiting...
```
instead of the usage description